etc/column/column-docker.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+[DEFAULT]
+# Log file
+log_file = /var/log/column/column.log
+
+# Log info level
+log_level = DEBUG
+
+# API config
+server = 0.0.0.0
```

Comment:
When you deploy in a container and you specify 0.0.0.0 you are only binding to the interfaces within the container. There is no use in binding to the 127.0.0.1 inside the container as you need to access the service from either other containers or from the docker host. The deployer will choose how he exposes the port as I mentioned in the commit message. He needs to specifically publish the port at runtime for it to be accessible on all of the host's interfaces. Hope this clears your apprehension.